cinema/tests/test_models.py

```diff
@@ -0,0 +1,410 @@
+import pytest
+
+from cinema.models import (
+    Country,
+    Genre,
+    ImdbRating,
+    MpaaRating,
+    Language,
+    Distributor,
+)
+
+country_names = ["USA", "UK", "France", "Italy", "Australia"]
+genre_names = ["Crime", "Drama", "Comedy", "Adventure", "Western"]
+imdb_ratings = [8.0, 8.5, 9.0]
+mpaa_ratings = ["PG-13", "R", "Not Rated"]
+mpaa_descriptions = [
+    "Children under 13", "Children under 17", "The upcoming movie"
+]
+language_names = ["English", "German", "Italian", "French", "Spanish"]
+distributor_names = [
+    "Columbia Pictures", "Paramount Pictures",
+    "Universal Pictures", "Warner Bros.", "Netflix"
+]
```

Comment:
Look, essentially you are introducing constants. It's clear. But in the pytest world, each of them will be a separate fixture that you will use in your tests. It is right.

```
@fixture
def country_names():
    return ("USA", "UK", "France", "Italy", "Australia",)
```

And never use list data type if you use constants and it never changing, always try to use a tuple. A tuple takes up at least half the space of a list.